sklearn/svm/base.py

```diff
@@ -604,30 +603,31 @@ def _get_liblinear_solver_type(multi_class, penalty, loss, dual):
       - loss
       - dual
 
-    The same number is internally by LibLinear to determine which
-    solver to use.
+    The same number is also internally used by LibLinear to determine
+    which solver to use.
     """
-
-    # nested dicts containing level 1: available loss functions, 
+    if loss.isupper() or penalty.isupper():
+        warnings.warn(("Case insensitivity support for loss and penalty"
+                       " parameters is deprecated from 0.16 and will be"
+                       " removed in 0.18. The values for loss and"
+                       " penalty are expected to be in lower case only."),
```

Comment:
> done ;)

This is not addressed in the last commit currently visible on github (635b933). Travis is still broken at this point.